charts/vela-minimal/templates/defwithtemplate/gateway.yaml

```diff
@@ -57,6 +66,10 @@ spec:
 
         	// +usage=Specify the class of ingress to use
         	class: *"nginx" | string
+
+        	// +usage=Set ingress class to 'kubernetes.io/ingress.class' annotation instead of spec.
+        	// Set to false if ingress controller supports '.spec.ingressClassName'.
+        	useClassAnnotation: *true | bool
```

Comment:
I think we can change `useClassAnnotation` to `classInSpec`, so the default value can be `false` which is more clear. WDYT?

Reply:
classInSpec is better

Reply:
I also think `classInSpec` is better. Thanks for your suggestion 🙂